Mage.Sets/src/mage/cards/l/LethalThrowdown.java

```diff
@@ -0,0 +1,95 @@
+package mage.cards.l;
+
+import java.util.List;
+import java.util.UUID;
+
+import org.checkerframework.checker.units.qual.s;
+
+import mage.abilities.Ability;
+import mage.abilities.costs.Cost;
+import mage.abilities.costs.common.DiscardCardCost;
+import mage.abilities.costs.common.SacrificeTargetCost;
+import mage.abilities.effects.common.DestroyTargetEffect;
+import mage.abilities.effects.common.DrawCardSourceControllerEffect;
+import mage.abilities.effects.OneShotEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Outcome;
+import mage.counters.Counters;
+import mage.filter.StaticFilters;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.target.common.TargetCreatureOrPlaneswalker;
+
+/**
+ *
+ * @author grimreap124
+ */
+public final class LethalThrowdown extends CardImpl {
+
+    public LethalThrowdown(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[] { CardType.SORCERY }, "{B}");
+
+        // As an additional cost to cast this spell, sacrifice a creature or sacrifice a modified creature.
+        this.getSpellAbility().addCost(new SacrificeTargetCost(StaticFilters.FILTER_PERMANENT_CREATURE)
```

Comment:
This is not correct. It is worded that way in order to allow the player to not draw if it is detrimental.

Reply:
What part of it isn't right? Don't they have to sacrifice either a creature or a modified creature to cast the spell?

Reply:
There is two alternatives to the additional cost. It is possible to sacrifice a modified creature to the first option (sacrifice a creature), in which case there is no card draw.

Reply:
I handle the alternative of the creature being modified in LethalThrowdownEffect.
But maybe I'm not understanding the problem

Reply:
On casting the spell, the player should have the choice between 'sacrifice a creature' and 'sacrifice a modified creature'. Then pay for that choice. A modified creature could be sacrificed for either.

On resolve, the draw a card part will only applied if the 'sacrificed a modified creature' cost has been paid.

Reply:
I see what you're saying, I'll update it soon

Reply:
I have pushed an update, I don't know how much you'll like it though. I couldn't find a way to get the selected cost of an OrCost and then nicely compare it to the different costs so I just used getText()